### ISSUE TITLE
[BUG] No Decky Menu after April 3rd Steam Beta Client Update

### ISSUE THREAD

**Issue by The1AndMany** [SteamDeckHomebrew/decky-loader#414]
### Please confirm

- [X] I have searched existing issues
- [X] This issue is not a duplicate of an existing one
- [X] I have checked the [common issues section in the readme file](https://github.com/SteamDeckHomebrew/decky-loader#-common-issues)
- [X] I have attached logs to this bug report (failure to include logs will mean your issue will not be responded too).

### Bug Report Description

After updating to the newest Steam Beta Client Update, I lost access to the Decky menu. My various CSS Loader settings are applying as usual, including the semi-transparent keyboard in Desktop Mode.

Steps to reproduce:
1. Update to the April 3rd Steam Beta Client Update in the Settings menu
2. Once the update is complete, press the [...] button to open the Quick Access Menu
3. No Decky menu tab in QAM

### Expected Behaviour

Decky loader menu should appear as a tab in the QAM, just below the "Help" tab.

### SteamOS version

SteamOS 3.4.6

### Selected Update Channel

Prerelease

### Logs

[deckylog.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/11150731/deckylog.txt)

**Comment by Logge1002** [comment]
Same for me. Hope this gets fixed soon. No vibrant deck is making me miss it so much

**Comment by PartyWumpus** [comment]
it is still working in stable, so for now you can just switch to stable steamos.

**Comment by Neablis** [comment]
Confirmed its missing when in prerelease and comes back when you switch back to stable. Had the same issue and this fixed it.

**Comment by Delikt** [comment]
+1 also broken Decky Loader since 3th April but i avoid switching to stable brunch cause there are finally a fix for the 4k Display Bug (loosing Display Connection) in the Preview/Beta Channel which is much more annoying as having a broken Decky Loader

**Comment by PartyWumpus** [comment]
Prerelease should now work if you want to stay on non stable steamos, but a handful of plugins haven't been fixed yet.
Known broken:
```
- Audio Loader (Fix incoming)
- Free Games
- MusicControl
- CSS Loader (Fix incoming)
- Animation Changer
- Flatpak Updater
```

**Comment by MrHappy** [comment]
Weird, I am on 3.4.6 and I do not see the option...
